src/data/graphql/queries.graphql

```diff
@@ -204,6 +204,21 @@ query UserTokens($address: String!) {
   }
 }
 
+query UserBalanceWithLock($address: String!, $tokenAddress: String!) {
+  user(address: $address) {
```

Comment:
Is `withLock` and further in the code `userLock`  inactive tokens? If yes, can we please rename it to `inactive`. In my popover I will actually have total locked tokens which you don't refer to here (as far as I understand), so I think it will be a bit confusing. 

I'm not trying to nitpick, I just think it will be easier to have the names corresponding to the UI: active, inactive, total, total locked (I am not sure about the last one as I still need to understand what it refers to).

Reply:
so this is the contract:
```  struct Lock {
    // User's lock count
    uint256 lockCount;
    // Deposited balance
    uint256 balance;
    // Weighted average of deposit timestamps (no longer used)
    uint256 DEPRECATED_timestamp; // solhint-disable-line var-name-mixedcase
    // Pending balance (from failed transfers), can claim with a forced deposit
    uint256 pendingBalance;
  }```
and this is the network method im calling: getUserLock

I would honestly prefer to be consistent in naming with network.
Balance - from Lock contract is the balance which is currently "stake or locked" for this user
balance from nativeToken - Is balance which this user have in this token
TOTAL balance = userLock.balance + userLock.nativeToken.balance
I wouldn't name userLock.balance = inactive - because from different perspective is not necessarily "inactive" is just "staked" or "locked"

Reply:
I see your point... All of these balances are not clear to me... Especially I know that we should show another one as per Karol's message in discord:
"these are still active tokens, however they are already in use somewhere. for example, being part of expenditure budget, at pending state, and so on. so as a result, they cannot be withdrawn. let me know if that sounds clear"

Can we maybe have a call where you explain the contract balances to me, please? Should we include Alex as well?